app/javascript/styles/mastodon/components.scss

```diff
@@ -1357,6 +1353,12 @@
   }
 }
 
+.account__avatar:not([width]) {
```

Comment:
This looks like a hack. What's going on here? If it has a width attribute, wouldn't it be an `img` tag then? And if so, what is `background-size` doing?

Reply:
it is, but so are the other cases we want that style to apply to. It's just that the only case that use a non-default 36px also set the `width` and `height` attributes.